### ISSUE TITLE
[Theme Request] Rei Ayanami (black plugsuit)

### ISSUE THREAD

**Issue by Madahin** [doki-theme/doki-master-theme#46]
## Who/What do you want to be featured?
Rei Ayanami from Neon Genesis Evangelion (but with her black plugsuit)

### What do they look like?
![8231280207_3af51b6829_b](https://user-images.githubusercontent.com/2446547/108234648-4c329e80-7145-11eb-8234-ff6298e12446.jpg)
![f0a1b77491a62f9653ddac776bce7123](https://user-images.githubusercontent.com/2446547/108234675-52287f80-7145-11eb-8521-2f2750c5545b.jpg)
![tumblr_o04izeMDlY1v39f3co1_1280](https://user-images.githubusercontent.com/2446547/108234687-55bc0680-7145-11eb-95b8-0695b22050b1.jpg)

## What type of theme?

- [ ] Light Theme
- [X] Dark Theme

## Additional Information (Optional)
My current favorite theme is [One Dark Vivid](https://plugins.jetbrains.com/plugin/11938-one-dark-theme) if this can help in what kind of colorscheme i'm looking for.

Also, since everyone seems to put a video here : https://youtu.be/46AXsvhIj18

**Comment by Unthrottled** [comment]
Wow! I did not know that this variant of Rei existed!

> My current favorite theme is One Dark Vivid if this can help in what kind of colorscheme i'm looking for.

Neat! Well if I am going to build this the color palette of the syntax coloring and editor look and feel will be chosen from Rei and her dark suit like below.

![Rei](https://user-images.githubusercontent.com/2446547/108234687-55bc0680-7145-11eb-95b8-0695b22050b1.jpg) 

(Which I do like very much!)

If you want [one-dark](https://plugins.jetbrains.com/plugin/11938-one-dark-theme) with Rei, might I suggest [setting the background image](https://www.jetbrains.com/help/idea/setting-background-image.html). (It would be faster for you!)

**Comment by Madahin** [comment]
> If you want one-dark with Rei, might I suggest setting the background image. (It would be faster for you!)

I... didn't think of this. It's a good solution, although i would obviously prefer a theme made around Rei.
I'll still keep the request open in case you ever make this theme (and also because doki theme need more NGE waifu)

> Wow! I did not know that this variant of Rei existed!

It's from the [third movie](https://en.wikipedia.org/wiki/Evangelion:_3.0_You_Can_(Not)_Redo) from the [Rebuild of Evangelion](https://en.wikipedia.org/wiki/Rebuild_of_Evangelion) reboot. I cannot recommend these movies enough !

**Comment by Unthrottled** [comment]
> (and also because doki theme need more NGE waifu

I've been wanting to add Asuka and Rei for a while, but Asuka is red and Rei is blue, and I already have a bunch of blue and red themes. 

I think this variant of Rei would be nice :ok_hand: 

Looks like I need to watch some Eva movies!

**Comment by Unthrottled** [comment]
Hey! Figured I'd share an update. I am pretty happy with how the theme has turned out. I was looking for a green and black theme. Green, red, and black looks soo good!

I got to look the way I wanted it to, and this will be the final product. (I'll probably change Rei's sticker to be looking directly at me, but that's the only change)

![Screenshot from 2021-03-25 06-34-01](https://user-images.githubusercontent.com/15972415/112466753-2f851a00-8d34-11eb-83ad-b3b044498260.png)
